src/node_file.cc

```diff
@@ -475,9 +478,9 @@ void FillStatsArray(double* fields, const uv_stat_t* s) {
   fields[9] = -1;
 #endif
   // Dates.
-#define X(idx, name)                                                          \
-  fields[idx] = (static_cast<double>(s->st_##name.tv_sec) * 1000) +           \
-                (static_cast<double>(s->st_##name.tv_nsec / 1000000));        \
+#define X(idx, name)                                                               \
+  fields[idx] = (static_cast<double>(s->st_##name.tv_sec) * MILLISEC_PER_SEC) +    \
+                (static_cast<double>(s->st_##name.tv_nsec) / NANOSEC_PER_MILLISEC);\
```

Comment:
Quite frankly I don't see why these macros are needed when they are really only going to be used once and whose meanings are clear from the context. For mistake prevention `1e3` and `1e6` may be used (which incidentally would actually render the `double` cast unnecessary since scientific notation represents a double in C++).